### PR TITLE
Fix install error in python3.7+

### PR DESCRIPTION
**Pull request by abersheeran**
https://github.com/ericvsmith/dataclasses/commit/44724c459fcca9ddeab77b8d79d0790bb1c3163e

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def find_version(*file_paths):
         'sqlparse==0.2.4',
         'pymongo>=3.2.0',
         'django>=2.0',
-        'dataclasses>=0.1',
+        'dataclasses==0.6',
```
Comment (jay20162016) [inline setup.py]: It should probably be:
`        'dataclasses>=0.6',`
for compatibility for future versions

Comment (abersheeran) [reply setup.py]: dataclasses>=0.7 only supports python3.6

Comment (jay20162016) [reply setup.py]: oh. didn't notice

Comment (jay20162016) [inline setup.py]: You should delete this line, dataclasses is included in python already.

Comment (abersheeran) [reply setup.py]: So what to do in python 3.6? I don't want to answer your stupid questions, please don't ask them again.

**Comment by abersheeran** [comment]
Maybe better PR at https://github.com/nesdis/djongo/pull/374

**Pull request closed without merging**